airbyte-integrations/connectors/source-github/unit_tests/test_source.py

```diff
@@ -165,3 +169,24 @@ def test_config_validation(repos_config, expected):
 def tests_get_and_prepare_repositories_config(config, expected):
     actual = SourceGithub._get_and_prepare_repositories_config(config)
     assert actual == expected
+
```

Comment:
I'd appreciate a test to check that the token rotation logic works correctly.

Reply:
I have improved unit test